Assets/Scripts/Player/PlayerWeaponManager.cs

```diff
@@ -0,0 +1,73 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerWeaponManager : MonoBehaviour
```

Comment:
I'm not the biggest fan of how this class is written because it's very fixed to a specific inventory layout, and because of that, you have to do things like below:

```
currentWeapon.SetActive(true);
pistol.SetActive(false);
shotgun.SetActive(false);
rocketLauncher.SetActive(false);
```

but let's get this out for beta

Reply:
agreed